src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/backlink/BackLink.scala

```diff
@@ -32,6 +33,12 @@ object BackLink {
 
   def defaultObject: BackLink = BackLink()
 
+  def mimicsBrowserBackButtonViaJavaScript(implicit messages: Messages): BackLink = BackLink(
```

Comment:
I'm assuming this would generally be used in a context where `Messages` are available, and we'd want to avoid having to explicitly pass in `content`?

Reply:
this sounds good to me, doing it this way matches the [current default](https://github.com/hmrc/play-frontend-hmrc/blob/main/src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcLayout.scala.html#L87) we use, and if they want to override something they can `.copy()` it

Reply:
I'm somewhat assuming with this that people won't want to have a fallback url as well, because if they have a real url to go back to then it would be better to not use this js behaviour anyway